test/system/src/test/java/io/pravega/test/system/ConsumptionBasedRetentionWithMultipleReaderGroupsTest.java

```diff
@@ -391,6 +414,123 @@ public void updateRetentionPolicyForCBRTest() throws Exception {
                 new StreamImpl(SCOPE_1, STREAM_2), 0L).join().values().stream().anyMatch(off -> off == 390));
     }
 
+    @Test
+    public void multipleControllerCBRTest() throws Exception {
+        // scale to three controller instances.
+        scaleAndUpdateControllerURI(3);
+        assertTrue("Creating scope", streamManager.createScope(SCOPE_2));
+        assertTrue("Creating stream", streamManager.createStream(SCOPE_2, STREAM_3, STREAM_CONFIGURATION));
```

Comment:
Can we create stream with more than one segment?

Reply:
This can be taken care of during stream scaling test.